es-core/src/Settings.cpp

```diff
@@ -16,6 +16,7 @@ std::vector<const char*> settings_dont_save = boost::assign::list_of
 	("ForceKiosk")
 	("IgnoreGamelist")
 	("HideConsole")
+	("System")
```

Comment:
If we want this setting to persist across sessions (power-down/ups), this needs to be removed.